bftengine/src/preprocessor/tests/preprocessor_test.cpp

```diff
@@ -769,7 +771,7 @@ TEST(requestPreprocessingState_test, batchCancelledNoConsensusReached) {
   }
   auto clientBatchReqMsg = make_unique<ClientBatchRequestMsg>(clientId, reqBatch, batchSize, cid);
   // Call the PreProcessor callback to handle ClientBatchRequest message
-  reqMsgHandlerCallback(move(clientBatchReqMsg));
+  reqMsgHandlerCallback(std::move(clientBatchReqMsg));
```

Comment:
This is good for future, in practice it does nothing - compiler will delete the move constructor since we already define custome copy constructir (rule of 5).
We have this issue on all messages actually which do the same.

same for all below new std::move

Reply:
I did not add 'move' now, just put 'std::' for readability.